src/main/drivers/accgyro_spi_mpu6500.c

```diff
@@ -163,8 +167,32 @@ static void mpu6500AccRead(int16_t *accData)
     accData[Z] = (int16_t)((buf[4] << 8) | buf[5]);
 }
 
+#ifdef COLIBRI_RACE
+static void mpu6500SpiInit(void)
```

Comment:
Should that not be added here?
https://github.com/cleanflight/cleanflight/blob/c142e3dfe988286d37e898940b0f772d8968db79/src/main/sensors/initialisation.c#L227-L244